container/dockershell.sh

```diff
@@ -57,6 +57,7 @@ OPTIONS:
                   e.g. "db2eventstore-1578174815082"
 --user         User name of the Event Store server
 --password     Password of the Event Store server
+--es-version   The Event Store version you want to install 
```

Comment:
I think this is a little confusing as it's not the eventstore version you want to install but rather:
`The event store version of the target cluster`

Reply:
I am not seeing the difference, but it might be not geting the signifigance of "target cluster" but can change it

Reply:
The point being `dockershell` and the `IOT application container` is not installing anything per se, rather it is setting up a client to point to an already installed eventstore cluster.

So the idea is to avoid any confusion that we are "installing eventstore" locally.